pkg/antctl/raw/migrate/checkenv/command_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checkenv
+
+import (
+	"testing"
+)
+
+func TestCheckSelector(t *testing.T) {
```

Comment:
What about other function's unit test coverage? The patch coverage of this PR is 8%, you need to improve the unit test.

Reply:
Sure, all conversion functions could be converted, I'm working on this .